server/controllers/article.js

```diff
@@ -259,3 +259,39 @@ export const deleteArticle = async (req, res) => {
     return serverError(res, error);
   }
 };
+
+export const rateArticle = async (req, res) => {
+  const { slug, user } = req;
+  const article = await Articles.findOne({ where: { slug } });
```

Comment:
we are using a middleware to find article, please endeavor to use the middleware

Reply:
It has been modified to cater for this scenario